modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -937,7 +946,7 @@
 			<return type="int">
 			</return>
 			<description>
-				Returns a random unsigned 32 bit integer. Use remainder to obtain a random value in the interval [code][0, N][/code] (where N is smaller than 2^32 -1).
+				Returns a random unsigned 32 bit integer. Use remainder to obtain a random value in the interval [code][0, N - 1][/code] (where N is smaller than 2^32 -1).
```

Comment:
I guess N should thus be smaller than `2^32`, not `2^32 - 1`?

Reply:
Might also reword it as `[0, N)` or `between 0 and N - 1, inclusive`.

Reply:
Yes, thank you for correcting that :)

Reply:
> Might also reword it as `[0, N)` or `between 0 and N - 1, inclusive`.

I thought about as well. But there are no `[a, b)` notations in the docs before. Only some `[a, b - 1]` exists.

Reply:
No I think `[a, b - 1]` is clearer. I'm not familiar with the `[a, b)` notation for exclusive, in school I learned `[a, b[` (instead `[a, b)` was used for half-lines going from `a` to the direction of `b` and beyond.

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/Interval_(mathematics)#Including_or_excluding_endpoints), both `[a, b[` and `[a, b)` are correct and standardized by ISO 31-11.